### PR TITLE
build: Add A380X build actions

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary of Changes

- Build jobs for A380X to build on monorepo for master and PRs
- Removed unused experimental workflow

Discord username (if different from GitHub): cdr_maverick

## Testing instructions
n/a

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/wasm/fadec_a320/build.sh [hunk 1]
```diff
@@ -12,9 +16,6 @@ else
   WASMLD_ARGS="-O2 --lto-O2 --strip-debug"
   CLANG_ARGS="-flto -O2 -DNDEBUG"
 fi
-
-set -ex
-
```
Comment (tracernz) [inline fbw-a32nx/src/wasm/fadec_a320/build.sh:17]: We need `set -e` here.

**Review by tracernz — CHANGES_REQUESTED** [review]

File: .github/workflows/master-a380x.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      - master
+
+jobs:
+  build:
+    # Prevent running this on forks
+    if: github.repository_owner == 'flybywiresim'
+    runs-on: ubuntu-latest
+    env:
+      A32NX_PRODUCTION_BUILD: 1
+      A32NX_INSTRUMENTS_BUILD_WORKERS: 2
```
Comment (MicahBCode) [inline .github/workflows/master-a380x.yml:20]: Do we need to use the environment variables with the A32NX prefix here or may we either introduce A380X specific environment variables or change these ones to a more general name?

File: .github/workflows/master-a380x.yml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        run: |
+          ./scripts/dev-env/run.sh ./scripts/setup.sh --clean
+          ./scripts/dev-env/run.sh ./scripts/build_a380x.sh --no-tty -j 4
+          rm -rf fbw-a380x/src
+          docker system prune -af
+      - name: Build ZIP files
+        run: |
+          df -h
+          node ./scripts/fragment_a380x.js
+          cp ./fbw-a380x/out/build-modules/modules.json ./fbw-a380x/out/flybywire-aircraft-a320-neo/install.json
```
Comment (MicahBCode) [inline .github/workflows/master-a380x.yml:52]: Is out/flybywire-aircraft-a320-neo right here?

File: .github/workflows/pr.yml [hunk 4]
```diff
@@ -74,8 +78,48 @@ jobs:
...
+    env:
+      A32NX_PRODUCTION_BUILD: 1
+      A32NX_INSTRUMENTS_BUILD_WORKERS: 2
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+      - name: Create .env file
+        # We do not set FBW_TYPECHECK=1 here, as there are still things to fix in the A380 instruments
+        run: |
+          echo A32NX_PRODUCTION_BUILD=1 >> fbw-a380x/.env
```
Comment (MicahBCode) [inline .github/workflows/pr.yml:99]: See environment variables comment .github/workflows/master-a380x.yml

File: .github/workflows/pr.yml [hunk 5]
```diff
@@ -74,8 +78,48 @@ jobs:
...
-          name: A32NX
+          name: flybywire-aircraft-a320-neo
           path: ./fbw-a32nx/out/flybywire-aircraft-a320-neo/
+  build-a380x:
+    name: Build A380X
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
+    env:
+      A32NX_PRODUCTION_BUILD: 1
+      A32NX_INSTRUMENTS_BUILD_WORKERS: 2
```
Comment (MicahBCode) [inline .github/workflows/pr.yml:92]: See environment variables comment .github/workflows/master-a380x.yml

File: .github/workflows/master-a380x.yml (hunk 2 above)
Comment (Benjozork) [reply .github/workflows/master-a380x.yml]: I renamed `A32NX_PRODUCTION_BUILD` to `FBW_PRODUCTION_BUILD`. Removed `A32NX_INSTRUMENTS_BUILD_WORKERS` as it's no longer used.

File: .github/workflows/master-a380x.yml (hunk 3 above)
Comment (Benjozork) [reply .github/workflows/master-a380x.yml]: I reworked this into  one `master.yml` workflow.

**Review by tracernz — APPROVED** [review]

**Pull request merged**